pvl-tools/tab/1.cson

```diff
@@ -274,61 +274,58 @@ services:
       , b: "username_service"
       , error: ["BAD_API_URL", "Bad hint from server; URL should contain username matching %{username_service}; got %{username_from_url}"] } },
-    # Create the mobile site url
+    # Create the 'oembed' url to fetch from
     { fill: {
-      , with: "https://mobile.twitter.com/%{username_from_url}/%{remainder}"
+      , with: "https://api.twitter.com/1/statuses/oembed.json?id=%{tweet_id}"
       , into: "our_url" } },
     # url validation passed
     { fetch: {
       , from: "our_url"
-      , kind: "html" } },
+      , kind: "json" } },
 
     # Check the author.
-    { selector_css: {
-      , selectors: [".main-tweet-container .main-tweet", 0, ".user-info-username", 0 ]
-      , into: "tweet_poster_1" # has a @ and extra newlines
-      , error: ["FAILED_PARSE", "Couldn't find a div .user-info-username"] } },
-    { whitespace_normalize: {
-      , from: "tweet_poster_1"
-      , into: "tweet_poster_2" } },
-    # Strip the '@' off the username
+    # Use author_url. Don't use author_name because that is sometimes a full name like 'Max Krohn'
+    { selector_json: {
+      , selectors: ["author_url"]
+      , into: "author_url"
+      , error: ["CONTENT_MISSING", "Could not find 'author_url' in json"] } },
     { regex_capture: {
-      , pattern: "^@?(.*)$"
-      , from: "tweet_poster_2"
-      , into: ["tweet_poster_3"]
-      , error: ["FAILED_PARSE", "Could not process username '%{tweet_poster_2}'"] } },
+      , pattern: "^https://twitter\\.com/(.+)$"
+      , from: "author_url"
+      , into: ["author_username"]
+      , error: ["CONTENT_MISSING", "Could not capture username from author_url"] } },
     # Check the username in the tweet. Case insensitive.
     { assert_compare: {
       , cmp: "cicmp"
-      , a: "tweet_poster_3"
+      , a: "author_username"
       , b: "username_service"
-      , error: ["BAD_USERNAME", "Bad post authored: wanted '%{username_service}' but got '%{tweet_poster_2}'"] } },
+      , error: ["BAD_USERNAME", "Bad post authored: wanted '%{username_service}' but got '%{author_username}'"] } },
 
-    { selector_css: {
-      , selectors: [".main-tweet-container .main-tweet", 0, ".tweet-content .tweet-text", 0 ]
+    { selector_json: {
+      , selectors: ["html"]
       , into: "tweet_contents"
-      , error: ["CONTENT_MISSING", "Missing <div class='tweet-text'> container for tweet"] } },
+      , error: ["CONTENT_MISSING", "Missing 'tweet_html' in json"] } },
     { whitespace_normalize: {
       , from: "tweet_contents"
       , into: "tweet_contents_nw" } },
-    # Strip mentions off the front. Get username and sig.
+    # Get username and sig from tweet.
     { regex_capture: {
-      , pattern: "^ *(?:@[a-zA-Z0-9_-]+\\s*)* *Verifying myself: I am ([A-Za-z0-9_]+) on (?:https?://)?Keybase\\.io[\\s\\p{Zs}]*\\. (\\S+) */.*$"
+      , pattern: "^.*Verifying myself: I am ([A-Za-z0-9_]+) on (?:https?://)?Keybase\\.io[\\s\\p{Zs}]*\\. (\\S+) */.*$"
```

Comment:
This bit is more permissive than before. It used to just strip off mentions. Now it will accept any prefix at all.

The actual response is this junk
```
  "html": "<blockquote class=\"twitter-tweet\"><p lang=\"de\" dir=\"ltr\">Verifying myself: I am max on Keybase.io. ZnBizHMA8RKSB598TaDtjlPlLKSEu1WuaT59 / <a href=\"https://t.co/ZxkeIpVbxt\">https://t.co/ZxkeIpVbxt</a></p>&mdash; Max Krohn (@maxtaco) <a href=\"https://twitter.com/maxtaco/status/433688676975927296?ref_src=twsrc%5Etfw\">February 12, 2014</a></blockquote>\n<script async src=\"https://platform.twitter.com/widgets.js\" charset=\"utf-8\"></script>\n",
```